frontend/javascripts/oxalis/view/action-bar/default-workflow.ts

```diff
@@ -0,0 +1,24 @@
+export default `predict:
```

Comment:
re file name: how about `default-predict-workflow-template.ts`. the folder location is not optimal, either, but I'd love to restructure a lot of files, anyway. maybe in another pr...

Reply:
> the folder location is not optimal, either,

Yeah, agreed. Maybe let's tackle this at the next hackathon or so?